resolvers.py

```diff
@@ -0,0 +1,335 @@
+                    final_result[obj["project_name"]] = [obj]
+
+            return {"ok": True, "error": "", "data": final_result}
+        except Exception as e:
+            return {"ok": False, "error": f"The error is -> {e} ", "data": []}
+
+
+class UniqueAccountsResolver:
+    def __call__(self, source, info, **kwargs):
+        try:
+            return {
+                "ok": True,
+                "error": "",
+                "data": [DEFAULT_ACCOUNT_TO_DISPLAY]
+                + list(
+                    models.MasterPodDetailsFact.objects.order_by()
+                    .values_list("account_name", flat=True)
+                    .distinct()
+                ),
+            }
+        except Exception as e:
+            return {"ok": False, "error": f"The error is -> {e} ", "data": []}
+
+
+class BenchAgingGrandTotalByCapabilityResolver:
+    def __call__(self, source, info, **kwargs):
+        # filter by duration
+        # group by capability and sum allocation_pending, nominated, bench
+        try:
+            qs = models.BenchAgingByCapability.objects.filter(duration="ALL_DURATION")
+
+            final_qs = (
+                qs.values("capability_name")
+                .order_by("capability_name")
+                .annotate(
+                    no_of_employees=Sum("available_employees")
+                    + Sum("nominated_employees")
+                    + Sum("allocation_pending_employees"),
+                    id=Window(expression=RowNumber(), order_by=[F("capability_name")]),
+                )
+            )
+
+            return final_qs
+        except models.BenchAgingByCapability.DoesNotExist:
+            return []
+
+
+class BenchAgingGrandTotalByBandResolver:
+    def __call__(self, source, info, **kwargs):
+        # filter by duration
+        # group by band and sum allocation_pending, nominated, bench
+        try:
+            qs = models.BenchAgingByBand.objects.filter(duration="ALL_DURATION")
+
+            final_qs = (
+                qs.values("band_name")
+                .order_by("band_name")
+                .annotate(
+                    no_of_employees=Sum("available_employees")
+                    + Sum("nominated_employees")
+                    + Sum("allocation_pending_employees"),
+                    id=Window(expression=RowNumber(), order_by=[F("band_name")]),
+                )
+            )
+
+            return final_qs
+        except models.BenchAgingByBand.DoesNotExist:
+            return []
+
+
+class BaseDataSourceResolverByCapability:
+    def __call__(self, source, info, **kwargs):
+        # params required
+        # 1. capability name (Java,...., Total, Other)
+        # 2. start date, end date
+        # 3. type of data (Available, Nominated, Allocation Pending, Total bench)
+        # 4. band name (Optional)
+
+        capability_name = kwargs.get("capability_name")
+        band_name = kwargs.get("band_name")
+        start_date = kwargs.get("start_date")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You are using the `get` method without a default argument to return the value of a key in a dictionary. We recommended that you use a default argument so that if the value for your key is not found, a default value is returned. If a default value is not provided and the key is not found, then `None` is returned.

[Learn more](https://docs.python.org/3/library/stdtypes.html#dict.get)

Similar issue at line number 224.